libretroshare/src/file_sharing/hash_cache.cc

```diff
@@ -117,8 +117,8 @@ void HashStorage::data_tick()
                 if(!mChanged)	// otherwise it might prevent from saving the hash cache
                 {
                     std::cerr << "Stopping hashing thread." << std::endl;
-                    shutdown();
-                    mRunning = false ;
+					askForStop();
```

Comment:
Yes I'm 100% agree, but this is preferable to follow the style when a file is already wrote.